src/main/kotlin/racingcar/Cars.kt

```diff
@@ -0,0 +1,20 @@
+package racingcar
+
+import racingcar.model.Car
+import racingcar.model.RaceCondition
+import racingcar.model.move
+
+/**
+ * 자동차들의 객체 관리
+ * */
+class Cars(raceCondition: RaceCondition) {
+    private var _carList = (0 until raceCondition.carCount).map { Car() }.toList()
+    val carList = _carList
```

Comment:
초기화를 하면 값이 변경되지 않아요!
```suggestion
      val carList : List<Car>
        get() = this._carList
```